src/main/java/Floda.java

```diff
@@ -0,0 +1,83 @@
+import java.util.Scanner;
+
+public class Floda {
+    public static void main(String[] args) {
+        String name = "Floda";
```

Comment:
Redundant code as it can be combined with the next line
eg. System.out.println("Hello! I'm Floda");